Toxygates/build.xml

```diff
@@ -73,10 +73,12 @@
     <!-- Set up WEB-INF/lib with jars -->
     <delete dir="${wardir}/WEB-INF/lib" failonerror="false" />
     <mkdir dir="${wardir}/WEB-INF/lib" />
+    <ivy:retrieve pattern="${wardir}/WEB-INF/lib/[artifact]-[revision].[ext]" conf="runtime" type="bundle,jar"/>
```

Comment:
Moving this <ivy:retrieve> to be sequentially prior to the <copy todir> just below allows the latter to copy the mlib/ jars (on manual.classpath) over the ivy retrieved jars.
I kept intermine-ws-java in ivy.xml even though we overwrite it, since we still need its dependencies.